docs/how-to-get-help.md

```diff
@@ -1,15 +1,65 @@
-## Onboarders
+## Co-pilot programme
 
-When your project has been approved by NHS England, we will agree an intensive support period with you when our experienced researchers and engineers will be made available to get you started.
+When your project has been approved by NHS England,
+we will agree with you an intensive programme of support with our experienced researchers, or *co-pilots*.
+You can find out more about the co-pilot programme [on our blog][the co-pilot programme].
 
-## Q&A Forum
+[the co-pilot programme]: https://www.bennett.ox.ac.uk/blog/2021/08/opensafely-co-pilot-programme-assisting-users-on-their-opensafely-journey/
 
-If you're outside the onboarding support period, or you're just trying OpenSAFELY out, your first port of call should be our [Q&A forum](https://github.com/opensafely/documentation/discussions) in GitHub. Before asking for help here, please search the [Q&A forum](https://github.com/opensafely/documentation/discussions) and these documentation pages to check that your query hasn’t already been addressed. Some of the most common issues that users face when using OpenSAFELY are listed in the [FAQ page](https://github.com/opensafely/documentation/discussions/445) on the Q&A.
+## Q&A forum
+
+If you're outside the co-pilot programme,
+then your first port of call should be to ask your question in the [Q&A forum][].
+Before asking your question,
+please search the [Q&A forum][] and the OpenSAFELY documentation,
```

Comment:
Missing URLs for the Q&A forum

Reply:
There are a couple of ways to add links in Markdown. I think you're familiar with the first:

```
The [Q&A forum](https://github.com/opensafely/documentation/discussions) is great.
```

But from your comments, I don't think you're familiar with the second:

```
The [Q&A forum][] is great.

[Q&A forum]: https://github.com/opensafely/documentation/discussions
```

I tend towards the second, because I think it makes it easier to read the text. If you don't want to take my word for it, then the links are added in the rendered version of the page: https://opensafely-documentation-pr-867.onrender.com/how-to-get-help/.

---

There's actually a third way to add links in Markdown. If you want to display a URL and convert it to a link, then:

```
You'll find OpenSAFELY Jobs at <https://jobs.opensafely.org/>.
```

Reply:
You might find <https://commonmark.org/help/> useful 🙂